go/client/cmd_simplefs_cancel_uploads.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2020 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+// CmdSimpleFSCancelUploads is the 'fs cancel-uploads' command.
+type CmdSimpleFSCancelUploads struct {
+	libkb.Contextified
+	path   keybase1.Path
+	filter keybase1.ListFilter
+}
+
+// NewCmdSimpleFSCancelUploads creates a new cli.Command.
+func NewCmdSimpleFSCancelUploads(
+	cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+	return cli.Command{
+		Name:         "cancel-uploads",
+		ArgumentHelp: "[path-to-folder]",
+		Usage:        "cancel all outstanding uploads in a folder",
+		Action: func(c *cli.Context) {
+			cl.ChooseCommand(&CmdSimpleFSCancelUploads{
+				Contextified: libkb.NewContextified(g)}, "cancel-uploads", c)
+			cl.SetNoStandalone()
+		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "a, all",
+				Usage: "display entries starting with '.'",
```

Comment:
Should probably make it clear that this only affects the displayed path, but everything being uploaded will be canceled regardless of this flag. Perhaps soemthing like "display entries starting with '.' in the confirmation"?